Casks/fig.rb

```diff
@@ -2,7 +2,7 @@
   version "1.0.49,270"
   sha256 :no_check
 
-  url "http://versions.withfig.com/fig@latest.dmg",
+  url "https://versions.withfig.com/fig@latest.dmg",
```

Comment:
Is there a reason why we shouldn't use the versioned download here?

```suggestion
    url "https://versions.withfig.com/fig%20#{version.before_comma}.dmg",
```

Would require adding the `sha256` value also.